thirdeye/thirdeye-frontend/app/pods/components/self-serve-config-group-table/template.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="form-group col-xs-12">
+  {{#if data.length}}
+    {{#bs-accordion as |acc|}}
```

Comment:
Can we change `acc` to something more understandable? It's unclear to me what `acc` is here.